tools/sgcspevaluatorcli/tools.go

```diff
@@ -0,0 +1,59 @@
+package sgcspevaluatorcli
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const name = "csp"
+
+const packageJSONContent = `{
+  "dependencies": {
+    "@einride/csp-evaluator-cli": "^1.0.0"
+  }
+}
+`
+
+func Validate(ctx context.Context, csp string) *exec.Cmd {
+	return Command(ctx, "validate", csp)
+}
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
+	toolDir := sg.FromToolsDir(name)
+	if err := os.MkdirAll(toolDir, 0o755); err != nil {
+		return err
+	}
+	packageJSON := filepath.Join(toolDir, "package.json")
+	if err := os.WriteFile(packageJSON, []byte(packageJSONContent), 0o600); err != nil {
+		return err
+	}
+	sg.Logger(ctx).Println("installing packages...")
```

Comment:
put this at the top of the function so any potential error returned above comes after the log line?